data/categories.txt

```diff
@@ -287,7 +287,7 @@ cs:3Banka|peníze
 sk:3Banka|peniaze
 nl:3Bank|geld
 zh-Hant:1銀行|錢
-pl:3Bank|pieniądze
+pl:3Placówka banku|bank|pieniądze
```

Comment:
I'd leave "Bank" as the first option here, since it's displayed in the place page and in the categories list, and it might be more understandable.